server/src/main/java/io/envoyproxy/controlplane/server/V3DiscoveryServer.java

```diff
@@ -41,14 +41,15 @@ public V3DiscoveryServer(
       List<DiscoveryServerCallbacks> callbacks,
       ConfigWatcher configWatcher) {
     this(callbacks, configWatcher, new DefaultExecutorGroup(),
-        new DefaultProtoResourcesSerializer());
+        new DefaultProtoResourcesSerializer(), new DefaultStartupConfigs());
   }
 
   public V3DiscoveryServer(List<DiscoveryServerCallbacks> callbacks,
```

Comment:
minor: perhaps we want to create a 3rd constructor for the new param

Reply:
added.

TBH, I am on the fence. When I make envoy-control changes in our repo adding a new argument in a method. always debate whether to create a new method to make it compatible with existing code, so I do not need to change much. But if I add, code will be more and more verbose